src/applications/vaos/services/appointment/transformers.js

```diff
@@ -164,7 +167,10 @@ function getConfirmedStatus(appointment, isPast) {
  * @returns {Object} Returns appointment datetime as moment object
  */
 function getMomentConfirmedDate(appt) {
-  if (isCommunityCare(appt)) {
+  if (
+    isCommunityCare(appt) &&
+    (appt.communityCare === undefined || appt.communityCare === null)
```

Comment:
Maybe we should turn this whole check into `if (appt.appointmentTime) {`? While this check is correct, I think it's somewhat confusing ("if this appt is community care but also not community care").

Reply:
Haha, It does sound confusing when you lay it out in plain English. I'm now checking against `app.timeZone` instead